test/runtests.jl

```diff
@@ -335,6 +337,11 @@ using Test
         @test isempty(facts)
     end
 
+    @testset "v33 file" begin
+        net33 = parse_network("testfiles/synthetic_data_v33.RAW")
+        @test net33 isa Network
```

Comment:
Needs tests for all the specific sections

then i think we're pretty much done for this PR 🤞